src/components/Banner/index.jsx

```diff
@@ -13,10 +16,22 @@ const Banner = () => {
     <img
       alt="banner"
       className="banner"
-      src={Config.banner}
+      src={
+        isMobile
+          ? 'https://acmucsd.s3-us-west-1.amazonaws.com/portal/static/banner_s.png'
+          : 'https://acmucsd.s3-us-west-1.amazonaws.com/portal/static/banner_l.png'
+      }
```

Comment:
Hmm not sure if this is the right way of doing this but we could keep the AWS banner link `https://acmucsd.s3-us-west-1.amazonaws.com/portal/static/` in the config file, import it at the top, and append banner_s or banner_l in the ternary rather than keeping the full link in here. This would allow us to reuse it over and over as well. Thoughts?

Reply:
That's definitely a cleaner option. I'll go ahead and switch.